examples/api-routes/pages/api/people/index.js

```diff
@@ -1,5 +1,14 @@
 import { people } from '../../../data'
 
 export default (req, res) => {
-  res.status(200).json(people)
+  if (req.method === 'POST') {
+    if (req.headers['content-type'].startsWith('application/json')) {
+      // req.body will be automatically parsed to JSON when the request's Content-Type is `application/json`
+      res.json(req.body)
```

Comment:
It's generally considered a bad practice to mirror input back into the response as it causes vulnerabilities, so I'd rather not display that in the example. Instead you can maybe return something else 👍

Reply:
Thank you for your suggestion. That totally makes sense. 

I messed up by `force-pushing` somewhere. I hope I fixed it and that the returned texts now make more sense than returning the `req.body`